src/bundle.rs

```diff
@@ -500,6 +501,13 @@ impl<V: DynamicUsage> DynamicUsage for Bundle<Authorized, V> {
 #[derive(Debug)]
 pub struct BundleCommitment(pub Blake2bHash);
 
+impl From<BundleCommitment> for [u8; 32] {
```

Comment:
```markdown
### Added
- `impl From<BundleCommitment> for [u8; 32]`
```

Reply:
What distinguishes how we choose between providing `as_bytes`, `to_repr`, or other such methods and providing a `From` impl? After all the recent discussions we've had on this issue, I'd like us to have a standard style guide to refer to.

Reply:
https://rust-lang.github.io/api-guidelines/checklist.html is what I generally refer to. Specifically:
- [Ad-hoc conversions follow `as_`, `to_`, `into_` conventions](https://rust-lang.github.io/api-guidelines/naming.html#c-conv)
- [Conversions use the standard traits `From`, `AsRef`, `AsMut`](https://rust-lang.github.io/api-guidelines/interoperability.html#c-conv-traits)